www/Media.js

```diff
@@ -149,9 +149,21 @@ Media.prototype.release = function() {
  * Adjust the volume.
  */
 Media.prototype.setVolume = function(volume) {
-    exec(null, null, "Media", "setVolume", [this.id, volume]);
+    if (device.platform == 'iOS'){
```

Comment:
Doesn't this require the device plugin to be installed?
Instead, you should use ``` if (cordova.platformId === 'ios') ...```